### ISSUE TITLE
Reminder

### ISSUE THREAD

**Issue by riya-17** [ritwik12/Virtual-Assistant#16]
Reminder which can help us schedule our work or remember special days and to be up to time.

**Comment by ritwik12** [comment]
@riya-17 Yes that will be a great feature.

**Comment by rmereddy** [comment]
Hi is this issue still open i would like to work on adding the reminder feature?

**Comment by ritwik12** [comment]
@rmereddy Yes it is open, you can work on it.

**Comment by vnksnkr** [comment]
How should the feature work ? Should the assistant run in background ? One way is to ask the assistant what is on the user's schedule on particular day and get relevant information.

**Comment by ritwik12** [comment]
@vnksnkr  yeah, It will mainly depend on the third party app that you would like to use for Reminder, It could be google calendar, Outlook calendar, MacOs calendar or any other open source tool that will have calendars.

Choice and approach is totally on the developer, Do let us know your ideas before going forward to get better help.

**Comment by vnksnkr** [comment]
I was thinking of something that runs locally,keeps track of scheduled events and informs the users of the events scheduled for the particular day whenever the program is run

**Comment by vnksnkr** [comment]
I was thinking about running the feature locally that keeps track of events and the informs the user when there are events on a particular day.Using the calendar api requires OAuth2.0..could you give me some more pointers on how to implement it?

**Comment by ritwik12** [comment]
@vnksnkr You can do that locally also. As you mentioned.

**Comment by vnksnkr** [comment]
@ritwik12 Should I add the feature as a function in functions.c?
